src/applications/hca/components/GetFormHelp.jsx

```diff
@@ -21,7 +21,8 @@ function GetFormHelp() {
           contact={CONTACTS.HELP_TTY}
         />
         <br />
-        Monday &#8211; Friday, 8:00 a.m. &#8211; 8:00 p.m. ET
+        Monday &#8211; Friday, 8:00 a.m. &#8211; 10:00 p.m. ET, and Saturday,
+        8:00 a.m. &#8211; 5:00 p.m. ET.
```

Comment:
For accessibility reasons, this should say "Monday through Friday, 8:00 am to 10:00 p.m. ET, and Saturday 8:00 a.m. to 5:00 p.m. ET" - see https://design.va.gov/content-style-guide/dates-and-numbers#times-and-time-zones